posix.mak

```diff
@@ -13,6 +13,8 @@
 #
 # make BUILD=debug unittest => builds all unittests (for debug) and runs them
 #
+# make DEBUGGER=ddd BUILD=debug std/XXXXX.debug => builds the module XXXXX and executes it in the debugger ddd
```

Comment:
Well, as mention in person before calling the Makefile again could be a nice trick.